src/ui/style/index.less

```diff
@@ -8,10 +8,16 @@
 @import './var.less';
 @import './antd-overwrite.less';
 
-html,
-body {
-  font-family: -apple-system, BlinkMacSystemFont, 'Helvetica Neue', Roboto,
-    'PingFang SC', 'Microsoft Yahei', sans-serif;
+html {
```

Comment:
You should add 'body' selector, otherwise the added font won't work.

Reply:
Really? Added.

Reply:
This generates the selectors like `html[lang="ja"], body[lang="ja"]`. Should we add `lang` to `body` as well as `html`?